src/navsat_transform.cpp

```diff
@@ -448,6 +449,58 @@ bool NavSatTransform::fromLLCallback(
   return true;
 }
 
+bool NavSatTransform::fromLLArrayCallback(
+  const std::shared_ptr<robot_localization::srv::FromLLArray::Request> request,
+  std::shared_ptr<robot_localization::srv::FromLLArray::Response> response)
+{
+  double altitude, longitude, latitude;
+  double cartesian_x {};
+  double cartesian_y {};
+  double cartesian_z {};
+  tf2::Transform cartesian_pose;
+  std::string utm_zone_tmp;
+  for (auto it = request->ll_points.begin(); it != request->ll_points.end(); ++it) {
+    //
+    altitude = it->altitude;
+    longitude = it->longitude;
+    latitude = it->latitude;
+    cartesian_x = 0.0;
+    cartesian_y = 0.0;
+    cartesian_z = 0.0;
+    //
+    if (use_local_cartesian_) {
+      gps_local_cartesian_.Forward(
+        latitude,
+        longitude,
+        altitude,
+        cartesian_x,
+        cartesian_y,
+        cartesian_z);
+    } else {
+
+      navsat_conversions::LLtoUTM(
```

Comment:
I thought I deprecated these functions in favor of geographilib. But I cannot confirm until next week.

Reply:
Yes, we should be consistent with whatever method we're using.

Reply:
Found it!

Looks like parts didn't make it into ROS2. The reasoning of using geographiclib directly instead of these navsat_transform functions is here: https://github.com/cra-ros-pkg/robot_localization/pull/627/files#r598612280